### PR TITLE
Adds foundational classes to enable an operator to create Inspections

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: CameraViewer/CameraViewer/Samples/CameraControlTool/Part.cs [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
-        {
-            this.partNumber = partNumber;
-        }
-
-        private void setDescription(String description)
-        {
-            this.description = description;
-        }
-    }
-}
```
Comment (kulakowskin) [inline CameraViewer/CameraViewer/Samples/CameraControlTool/Part.cs:39]: Why did you delete the Part class?

Comment (ghost) [inline CameraViewer/CameraViewer/Samples/CameraControlTool/Part.cs:39]: I changed it to EnginePart. Should be under the changes tab
